website/docs/r/managed_application.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "Managed Applications"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_application"
+description: |-
+  Manages a Managed Application.
+---
+
+# azurerm_managed_application
+
+Manages a Managed Application.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_role_definition" "builtin" {
+  name = "Contributor"
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_managed_application_definition" "example" {
+  name                = "example-managedapplicationdefinition"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  lock_level          = "ReadOnly"
+  package_file_uri    = "https://github.com/Azure/azure-managedapp-samples/raw/master/Managed Application Sample Packages/201-managed-storage-account/managedstorage.zip"
+  display_name        = "TestManagedAppDefinition"
+  description         = "Test Managed App Definition"
+
+  authorization {
+    service_principal_id = data.azurerm_client_config.current.object_id
+    role_definition_id   = split("/", data.azurerm_role_definition.builtin.id)[length(split("/", data.azurerm_role_definition.builtin.id)) - 1]
+  }
+}
+
+resource "azurerm_managed_application" "example" {
+  name                        = "example-managedapplication"
+  location                    = azurerm_resource_group.example.location
+  resource_group_name         = azurerm_resource_group.example.name
+  kind                        = "ServiceCatalog"
+  managed_resource_group_name = "infrastructureGroup"
+  application_definition_id   = azurerm_managed_application_definition.example.id
+
+  parameters = {
+    location                 = azurerm_resource_group.example.location
+    storageAccountNamePrefix = "storeNamePrefix"
+    storageAccountType       = "Standard_LRS"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Managed Application. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Managed Application should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `kind` - (Required) The kind of the managed application. Possible values are `MarketPlace` and `ServiceCatalog`. Changing this forces a new resource to be created.
```

Comment:
```suggestion
* `kind` - (Required) The kind of the managed application to deply. Possible values are `MarketPlace` and `ServiceCatalog`. Changing this forces a new resource to be created.
```

Reply:
"to deply" - > "to deploy", right? If yes, I updated